conf/modules.config

```diff
@@ -126,8 +126,8 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
-    // CALL_SV_MANTA
-    withName: MANTA {
+    // CALL_STRUCTURAL_VARIANTS
+    withName: ".*CALL_STRUCTURAL_VARIANTS:CALL_SV_MANTA:MANTA" {
```

Comment:
I am not very familiar with this syntax, but does including the subworkflow name make it more specific to the process?

Reply:
Yup! It's helpful to use this syntax for the selector when you want multiple processes within the same subworkflow. For this one specifically, I thought it's better to be explicit than implicit 🤷 . We can change laterz

Reply:
Cool. What does the .* do?

Reply:
@jemten The `.*` globs the preceding stuff so this:
`NFCORE_RAREDISEASE:RAREDISEASE:CALL_STRUCTURAL_VARIANTS:CALL_SV_MANTA:MANTA`

Reply:
Ah!

Reply:
Sure! Makes sense 👍🏻